src/test/java/org/gridsuite/study/server/SecurityAnalysisTest.java

```diff
@@ -192,6 +194,7 @@ public void setup() throws IOException {
             .specificParametersPerProvider(Map.of())
             .build();
         String loadFlowParameters = objectMapper.writeValueAsString(loadFlowParametersInfos);
+        LIMIT_TYPE_JSON = objectMapper.writeValueAsString(List.of("CURRENT", "HIGH_VOLTAGE"));
```

Comment:
I think it would be clearer and more robust to directly use the enum here : 
```suggestion
        LIMIT_TYPE_JSON = objectMapper.writeValueAsString(List.of(
                LimitViolationType.CURRENT.name(),
                LimitViolationType.HIGH_VOLTAGE.name()));
```